### ISSUE TITLE
Can't Toggle On/Off per-page

### ISSUE THREAD

**Issue by Enkouyami** [lunu-bounir/allow-right-click.html#11]
Browser: Opera Version:73.0.3856.344
OS: Linux Mint 20 (x86_64; MATE)

After enabling the addon on a webpage by clicking the button, clicking the button again to disable it gives this error message:
> Allow right-click is already installed. If you still have issues with the right-click context menu, please use the FAQs page to report!

**Comment by lunu-bounir** [comment]
That's right. There is no toggle off feature yet. At the moment you need to refresh the page if you don't need the protection anymore.

**Comment by enochterror** [comment]
The 'add0n allow-right-click' extension removes a few event listeners on activation, overwriting specific methods native to the target site.

It can be inferred that the function does not Toggle, rather it deletes or disables Java items relevant to blocking context-menu.

Never was it implied , that you could just un-delete something.
